src/gui/accountsettings.cpp

```diff
@@ -376,10 +376,10 @@ void AccountSettings::slotMarkSubfolderEncrypted(FolderStatusModel::SubFolderInf
 
 void AccountSettings::slotEditCurrentIgnoredFiles()
 {
-    Folder *f = FolderMan::instance()->folder(selectedFolderAlias());
-    if (!f)
+    const auto folder = FolderMan::instance()->folder(selectedFolderAlias());
+    if (!folder)
```

Comment:
this is missing `{ }`

Reply:
I added braces to all the if statements missing them